### DISCUSSION TITLE
User Custom Presence

### DISCUSSION THREAD

**Discussion by manucabral** [manucabral/RichPresencePlus#9] [Ideas]
In addition to the application supporting multiple dynamically executed presences, the plan is to implement a customizable presence where the user can modify it to their preferences. While this customization won't be dynamic, it may be possible to implement in future iterations.

![image](https://github.com/user-attachments/assets/8800f588-f3d9-47df-a09f-341971c1bf51)

**Comment by larayavrs** [comment]
Agree

**Comment by manucabral** [comment]
![image](https://github.com/user-attachments/assets/e4217987-8e07-4b25-9627-519e89ad6358)

an example

**Reply by manucabral** [reply]
Started development in issue #17
